README.md

```diff
@@ -57,6 +57,44 @@ Build the helpers you want to use (you'll also need the corresponding language i
 
 If you run into any trouble with the above please create an issue!
 
+#### Running script with Docker
+
+If you don't want to setup the machine where the script will be executed, you could run the script within
+a `dependabot/dependabot-core` container.
+In order to do that, you'll have to pull the image from Docker Hub and mount your working directory into the container.
+You'll also have to set several environment variables to make the script work with your configuration, 
+as specified in the documentation.
+
+Steps:
+```shell
+$ docker pull dependabot/dependabot-core
+$ docker run -ti -w /home/dependabot/dependabot-script --env-file env.list -v "$(pwd):/home/dependabot/dependabot-script" dependabot/dependabot-core
+
+(Now you'd be inside the Docker container)
```

Comment:
Prefer running command outside of the container. Entering the container is considered as advanced usage. 

This also let you specify that the install step is required only once. 
For example:

1. Pull the Docker image
```shell
docker pull dependabot/dependabot-core
```

2. Install dependencies
```shell
docker run --rm dependabot/dependabot-core bundle install -j 3 --path vendor
```

3. Run dependabot
```shell
docker run --rm dependabot/dependabot-core -v ./:working-dir -e PROJECT=foo/bar bundle exec ruby ./script
```

Reply:
I've addressed your comments @LeoColomb.
Let me know if it looks ok to you.